packages/react-instantsearch/src/components/List.js

```diff
@@ -134,6 +134,7 @@ class List extends Component {
               selectItem(items[0], this.resetQuery);
             }
           }}
+          showLoadingIndicator={false}
```

Comment:
We could provide a default props for `showLoadingIndicator` with the value `false`. It will avoid the need to force the value to `false` when we don't want the indicator to display.